exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
Oh, I see what you mean -- the `updateFrameIndex` action updates the frameIndex property of the `participate.survey.index` controller and `this.set('frameIndex', frameIndex + 1);` on L150 is incrementing the exp-player's `frameIndex` property.